src/javascripts/components/tabs.js

```diff
@@ -14,7 +14,7 @@ var headingItemJsClass = 'js-tabs__heading'
 var tabContainerHiddenClass = 'app-tabs__container--hidden'
 var tabContainerJsClass = '.js-tabs__container'
 var tabContainerNoTabsJsClass = 'js-tabs__container--no-tabs'
-var allTabTogglers = '.' + tabsItemJsClass + ' a, ' + '.' + headingItemJsClass + ' a'
+var allTabTogglers = '.' + tabsItemJsClass + ' a, ' + '.' + headingItemJsClass + ' button,' + '.' + headingItemJsClass + ' a'
```

Comment:
@vanitabarrett Thanks for making those changes 👍  I just realised that we don't even need to get `headingItemJsClass + ' button'` on line 17 as they don't exist at that point 🙈 

Do you think it would work to move [line 91](https://github.com/alphagov/govuk-design-system/pull/1728/files#diff-e7f290d2691b8eae1a679d8335f4c2aa4f8e8af3e510e369cb7817c6c6c40f38R91) to line 18 and then just passing that variable to `enhanceMobileButtons()`? That way we'd grab all the DOM elements at the start of the file.

Reply:
Thanks @hannalaakso, good spot! I think I added that as part of the other approach, before we switched to only enhancing when JS is available, and it got left behind by mistake. I've removed the bit that's no longer needed and moved line 18 up to the top so we pass the mobile tabs through to `enhanceMobileButtons` now.